### PR TITLE
Add security and contributing files and some badges

### PR DESCRIPTION
**Pull request by mo-gr**
This adds security and contributing.md inspired by [skipper](https://github.com/zalando/skipper). Also a badge for license and travis status

### CODE REVIEW

File: .zappr.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+X-Zalando-Team: "pathfinder"
```
Comment (grassator) [inline .zappr.yaml]: I think for opensource we need more:

https://github.com/zalando/patroni/blob/master/.zappr.yaml

Comment (mo-gr) [inline .zappr.yaml]: added...

**Comment by mo-gr** [comment]
:+1:

**Comment by grassator** [comment]
👍

File: .zappr.yaml (hunk 1 above)
Comment (grassator) [inline .zappr.yaml]: Now we actually need to enable it :)

**Pull request merged**